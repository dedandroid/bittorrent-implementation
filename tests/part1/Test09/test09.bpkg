ident:6f92efa0b2fa66e752c8ffb6deabfeb5d47b31ce3b9927079df8caea885760d354f86b79603873f10e9d98710c97c6a1a9a041ab1d38039113eab5cb152dbdf5696ba06bd93dcdee0c8b146299046fac80720ddd70a3d91d5a87ef9889ce86a06226ff360ead7bbc53342dca688ff3f552c41fb1d5f40a16d4a016a8e37d663b8ffaeabcfa00416260276cf4f625d50540f2abea5faa00c60ee6adaa4cf11065068a16465e06ecceaa458ffcbedde32e9a9a1d16c1cae9983ddbcc78b459878121c3ed9ae65cfe52b2e8f50aa532cb3ecf1bcb5b1a708c24f0ce6c801b2d75c35de184c9e4970cbfc7d235241418adbfba03eddc16312e1f5e1844c58ed2b81622a0f7d0d010f2f410d549ad77f3f1923322af28038f673770cca691d84c9eec11ec00404cfa331a3d6d3ff083c11be3ccfcd3d1fcb2fcc3930c2fca38c43a7f66c399485fa5b784a80c8c7b53f8d783d477dbf2ba761fbb8b708ebd1a5e1d2f196e451ff86081b0c3041b1357174366cc411501e6167d6406811a4616d593b6f0715723e3950f905226c6ddca26eecde3e3a07840e4075597c5d92a35013cf1fd49eb22c06c7b112d6069aaeab16a2586e6208e0ea8ab9c9fcf86960576f9b70ad2a50b4a3e5caeb7e4d7aec24cb03ba1d46dfa728de3baa9e7185daa96ad14ef84c8f4a719ac3461c79144bea5bb99a2d7acb44cde803e1f5b0afc89134bc
filename:test09.data
size:512
nhashes:3
hashes:
	026cd08c852b5e9cc4995124d679f8bd826cf07a5f4d42536c7acd91df16d165
	9a8fbe29ba6742b8065e67a10b1c8b0eaa8e669b5c11b449ec7c6ce84ce86b6d
	9aa27a9b1d699198ac6c6583bcbffbcf9b54b3611a875fdc7e63ef48babd6288
nchunks:4
chunks:
	0ef5c50cc0818de19fc91910afc5ff2ea5652f7350bdba9d6e190e9b071c6634,0,128
	7bf13d6a3af9926b3e30d8474635a4d6db44fe77acd9c9586395ece9f3018755,128,128
	e4cd1112359947df5aba836557774f67a828748b4a27ab9475c95ca63c255900,256,128
	ca26523d1798e568712a005bee60b1383aba0498e0c26a578b285e408493c7ee,384,128
