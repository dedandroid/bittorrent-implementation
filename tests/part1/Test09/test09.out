7bf13d6a3af9926b3e30d8474635a4d6db44fe77acd9c9586395ece9f3018755
e4cd1112359947df5aba836557774f67a828748b4a27ab9475c95ca63c255900
